modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -13,21 +13,23 @@ def index
         use_cache = params[:useCache] || true
         start_date = params[:startDate] || one_year_ago.iso8601
         end_date = params[:endDate] || one_year_from_now.iso8601
-        page = params[:page] || { number: 1, size: 10 }
         reverse_sort = !(params[:sort] =~ /-startDateUtc/).nil?
 
         validated_params = Mobile::V0::Contracts::GetPaginatedList.new.call(
           start_date: start_date,
           end_date: end_date,
-          page_number: page[:number],
-          page_size: page[:size],
+          page_number: params.dig(:page, :number) || 1,
```

Comment:
I think it could be worth moving these defaults into the pagination helper. Currently, these are required here because I made them required in `GetPaginatedList`. This might be evidence against making them required.

Reply:
As mentioned in the comment below, I think moving these defaults into the pagination helper is the approach to take.